src/status_im/contact/db.cljs

```diff
@@ -120,6 +120,10 @@
                  (assoc % :admin? true)
                  %)))))
 
+(defn contact-exists?
+  [db public-key]
+  (some? (get-in db [:contacts/contacts public-key])))
```

Comment:
I don't think you need to use `some?` here, would work the same without if used for truthy, falsey